scorer/runner.py

```diff
@@ -1,24 +1,36 @@
+import os
 import time
+import zipfile
 import socket
 import traceback
-import os
 from datetime import datetime
 from arango import ArangoClient
+from py_expression_eval import Parser
+from hashlib import sha256
+import base64
 import config
+from verifications import yekta
 from verifications import seed_connected
 from verifications import brightid
+from verifications import seed_connected
 from verifications import dollar_for_everyone
-from verifications import yekta
 from verifications import seed_connected_with_friend
-from py_expression_eval import Parser
 
 db = ArangoClient(hosts=config.ARANGO_SERVER).db('_system')
+snapshot_db = ArangoClient().db('snapshot')
+verifiers = {
+    'SeedConnected': seed_connected,
+    'SeedConnectedWithFriend': seed_connected_with_friend,
+    'Yekta': yekta,
+    'BrightID': brightid,
+    'DollarForEveryone': dollar_for_everyone
+}
 
 
 def process(fname):
-    for verifier in [seed_connected, seed_connected_with_friend, yekta, brightid, dollar_for_everyone]:
+    for verification_name in verifiers:
         try:
-            verifier.verify(fname)
+            verifiers[verification_name].verify(fname)
```

Comment:
update `anti_sybil` to enable it loading graph from `snapshot` database instead of its backup file. So we will not need to pass backup file to verifiers anymore.

Reply:
It's done.